source/installing-splunk/splunk-distributed.rst

```diff
@@ -70,50 +73,204 @@ Each instance can be installed on different hosts following the same steps descr
+  [monitor:///var/ossec/logs/alerts/alerts.json]
+  disabled = 0
+  host = MANAGER_HOSTNAME
+  index = wazuh
+  sourcetype = wazuh
 
-  a) Download and insert the ``inputs.conf`` template to configure where the data will come from:
+  Now, following with the **outputs.conf**:
 
-    .. code-block:: console
+.. code-block:: console
+
+  # touch /opt/splunkforwarder/etc/system/local/outputs.conf
+
+And paste this inside:
+
+.. code-block:: xml
+
+  [indexer_discovery:cluster1]
+  pass4SymmKey = changeme
+  master_uri = https://<master_ip>:<port>
+
+  [tcpout:cluster1_tcp]
+  indexerDiscovery = cluster1
+
+  [tcpout]
+  defaultGroup = cluster1_tcp
+
+For the last one, the **props.conf**, follow the same procedure:
+
+.. code-block:: console
+
+  # touch /opt/splunkforwarder/etc/system/local/props.conf
+
+.. code-block:: xml
+
+  [wazuh]
+  DATETIME_CONFIG =
+  INDEXED_EXTRACTIONS = json
+  KV_MODE = none
+  NO_BINARY_CHECK = true
+  category = Application
+  disabled = false
+  pulldown_type = true
+
+To save all the changes, restart splunk:
+
+.. code-block:: console
+
+  # /opt/splunkforwarder/bin/splunk restart
+
+
+Indexers:
+_________
+
+Here, in the **master instance**, make the configuration that will be pushed to the indexers.
+
+For this configuration is necessary to create the following two files and paste into them the following blocks of code respectively:
+
+.. code-block:: console
+
+  # touch /opt/splunk/etc/master-apps/_cluster/local/inputs.conf
+
+.. code-block:: xml
+
+  [splunktcp://9997]
+  connection_host = ip
+
+
+.. code-block:: console
+
+  # touch /opt/splunk/etc/master-apps/_cluster/local/indexes.conf
+
+.. code-block:: xml
+
+  [wazuh]
+  coldPath = $SPLUNK_DB/wazuh/colddb
+  enableDataIntegrityControl = 1
+  enableTsidxReduction = 1
+  homePath = $SPLUNK_DB/wazuh/db
+  maxTotalDataSizeMB = 512000
+  thawedPath = $SPLUNK_DB/wazuh/thaweddb
+  timePeriodInSecBeforeTsidxReduction = 15552000
+  tsidxReductionCheckPeriodInSec =
 
-      # curl -so /opt/splunk/etc/system/local/inputs.conf https://raw.githubusercontent.com/wazuh/wazuh/3.7/extensions/splunk/peer-inputs.conf
+  [wazuh-monitoring-3x]
+  coldPath = $SPLUNK_DB/wazuh-monitoring-3x/colddb
+  enableDataIntegrityControl = 1
+  enableTsidxReduction = 1
+  homePath = $SPLUNK_DB/wazuh-monitoring-3x/db
+  maxTotalDataSizeMB = 512000
+  thawedPath = $SPLUNK_DB/wazuh-monitoring-3x/thaweddb
+  timePeriodInSecBeforeTsidxReduction = 15552000
+  tsidxReductionCheckPeriodInSec =
 
-  b) Download and insert the ``indexes.conf`` template to configure the indexes:
+
+Now, restart Wazuh:
+
+.. code-block:: console
+
+  # splunk restart
+
+.. note::
+
+  Check the state of the cluster executing:
 
     .. code-block:: console
 
-      # curl -so /opt/splunk/etc/system/local/indexes.conf https://raw.githubusercontent.com/wazuh/wazuh/3.7/extensions/splunk/peer-indexes.conf
+      # splunk show cluster-bundle-status
 
-  And finally, we need to restart the search peer:
 
```

Comment:
I think it will be better to explain the following block into the next section of the documentation, where the app installation is explained.